download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import logging
```

Comment:
what was the reason for adding `logging`? It doesn't look like you actually switched all the `print` statements everywhere, which means we now just have 2 inconsistent ways of logging instead of one

Reply:
The logging is easier in controlling the level of the messages to be displayed.  But changing all the prints to logging is too distractive in this PR.

Reply:
but since you are only using logging.info in this file I think its okay to just keep with print for consistency

Reply:
The logger is only used for once. I will change it back to print.